admin-dev/themes/new-theme/js/pages/country/index.ts

```diff
@@ -32,7 +32,6 @@ import BulkActionCheckboxExtension from '@components/grid/extension/bulk-action-
 import SubmitBulkExtension from '@components/grid/extension/submit-bulk-action-extension';
 import SubmitRowActionExtension from '@components/grid/extension/action/row/submit-row-action-extension';
 import LinkRowActionExtension from '@components/grid/extension/link-row-action-extension';
-import FormSubmitButton from '@components/form-submit-button';
```

Comment:
Isn't this needed? Seems like it's used to display a confirm modal when a for button is clicked, for delete button confirmation maybe?

Reply:
Well now we only need this button in form page and its in another ts file. Maybe its going to be used later when i add delete logic. But now its not needed so thats why it was deleted.

Reply:
Ah ok it was used in pages which mix the form and list JS code. Then good call it's better to keep both separated